doc/develop/test/twister.rst

```diff
@@ -459,15 +459,17 @@ harness: <string>
     - pytest
     - gtest
     - robot
-
-    Harnesses ``ztest``, ``gtest`` and ``console`` are based on parsing of the
-    output and matching certain phrases. ``ztest`` and ``gtest`` harnesses look
-    for pass/fail/etc. frames defined in those frameworks. Use ``gtest``
-    harness if you've already got tests written in the gTest framework and do
-    not wish to update them to zTest. The ``console`` harness tells Twister to
-    parse a test's text output for a regex defined in the test's YAML file.
-    The ``robot`` harness is used to execute Robot Framework test suites
-    in the Renode simulation framework.
+    - cpputest
+
+    Harnesses ``ztest``, ``gtest``, ``cpputest`` and ``console`` are based on parsing of the
+    output and matching certain phrases. ``ztest``, ``gtest`` and ``cpputest`` harnesses look
+    for pass/fail/etc. frames defined in those frameworks. Use ``gtest`` or ``cpputest``
+    harness if you've already got tests written with either of these framework and do
+    not wish to update them to zTest. If using ``cpputest`` be sure to pass the
+    verbose argument ``-v`` to `CommandLineTestRunner::RunAllTests`.
```

Comment:
Why the need for "-v"? I don't see it in the code. Also, it shouldn't be the case. Level of output verbosity mustn't affect a test result.

Reply:
The verbose argument must be applied to your  CPPUtest application. Correct me if I am wrong but if you do not add the verbose option, you do not get information about which test cases have passed. Instead you will only get a dot for each test that has passed (e.g. `....`) and at the end something such as ` OUTPUT: OK (2 tests, 2 ran, 94 checks, 0 ignored, 0 filtered out, 2 ms)`.